### PR TITLE
update wss4j project names

### LINKED ISSUE #19156: do not include version numbers in ws-security related project/bundle names
improvements

### PR DESCRIPTION
**Pull request by arunavemulapalli**
- remove version from bundle and projects
- update the references
resolves #19156

### CODE REVIEW

**Comment by arunavemulapalli** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.wssecurity_fat.wsscxf.1,com.ibm.ws.wssecurity_fat.wsscxf.2,com.ibm.ws.wssecurity_fat.wsscxf.3,com.ibm.ws.wssecurity_fat.wsscxf.4,com.ibm.ws.wssecurity_fat.wsscxf.saml.1,com.ibm.ws.wssecurity_fat.wsscxf.saml.2,com.ibm.ws.wssecurity_fat.wsscxf.saml.3,com.ibm.ws.wssecurity_fat.wsscxf.saml.5,com.ibm.ws.wssecurity_fat.wsscxf.saml.6

File: dev/com.ibm.ws.org.apache.wss4j.policy/bnd.overrides [hunk 1]
```diff
@@ -11,14 +11,14 @@
...
 Bundle-Description: org.apache.wss4j.wss4j.policy; version=2.3.0
-Bundle-SymbolicName: com.ibm.ws.org.apache.wss4j.policy.2.3.0
+Bundle-SymbolicName: com.ibm.ws.org.apache.wss4j.policy
 Bundle-ActivationPolicy: lazy
 
 WS-TraceGroup: WSS4J
 
 Export-Package: \
- org.apache.wss4j.policy.*
+ org.apache.wss4j.policy.*;version="2.3.0"
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.wss4j.policy/bnd.overrides]: Could we use this variable `bVersion` here so we only have to update the variable if we change the version export values?

File: dev/com.ibm.ws.org.apache.wss4j.policy/bnd.overrides [hunk 2]
```diff
@@ -11,14 +11,14 @@
 #     IBM Corporation - initial API and implementation
 #*******************************************************************************
 -include= ~../cnf/resources/bnd/rejar.props
-bVersion=1.0
+bVersion=2.3.0
 
 Bundle-Name: org.apache.wss4j.wss4j.policy
 Bundle-Description: org.apache.wss4j.wss4j.policy; version=2.3.0
```
Comment (neuwerk) [inline dev/com.ibm.ws.org.apache.wss4j.policy/bnd.overrides:17]: Could potentially use the `bVersion` variable here too right?

**Review by neuwerk — COMMENTED** [review]
Other than changing the bnd files to take advantage of the `bVersion` variable the changes look good to me.

**Review by neuwerk — DISMISSED** [review]
LGTM, thanks @arunavemulapalli

**Comment by arunavemulapalli** [comment]
#build

**Comment by arunavemulapalli** [comment]
#libby

**Comment by arunavemulapalli** [comment]
#libby

**Review by neuwerk — DISMISSED** [review]
LGTM!

**Review by neuwerk — APPROVED** [review]
Thanks Aruna!

**Comment by arunavemulapalli** [comment]
wssecurity tests results look good - https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_b-Ok0AOeEe6Gnsx4pF5sWA&tab=com.ibm.team.build.web.ui.internal.editors.result.activity.ActivitiesProvider

**Pull request merged**